lib/msf/core/auxiliary/scanner.rb

```diff
@@ -299,6 +299,36 @@ def scanner_show_progress
   end
 end
 
+def add_delay_jitter(_delay, _jitter)
+  # Introduce the delay
+  delay_value = _delay.to_i
```

Comment:
- ```add_delay_jitter``` function name is easily misunderstood. A better name may exist.
- ```to_i``` can be removed as what @jhart-r7 said.

```
msf auxiliary(tcp) > show options 

Module options (auxiliary/scanner/portscan/tcp):

   Name         Current Setting  Required  Description
   ----         ---------------  --------  -----------
   CONCURRENCY  10               yes       The number of concurrent ports to check per host
   DELAY        -1               yes       The delay between connections, per thread, in milliseconds
   JITTER       -1               yes       The delay jitter factor (maximum value by which to +/- DELAY) in milliseconds.
   PORTS        80               yes       Ports to scan (e.g. 22-25,80,110-900)
   RHOSTS       192.168.1.1      yes       The target address range or CIDR identifier
   THREADS      1                yes       The number of concurrent threads
   TIMEOUT      1000             yes       The socket connect timeout in milliseconds
msf auxiliary(tcp) > set DELAY test
DELAY => test
msf auxiliary(tcp) > run 
[-] Auxiliary failed: Msf::OptionValidateError The following options failed to validate: DELAY.

msf auxiliary(tcp) > run 

[-] Auxiliary failed: Msf::OptionValidateError The following options failed to validate: JITTER.
[*] Auxiliary module execution completed
```